test/functional/Java8andUp/playlist.xml

```diff
@@ -1936,6 +1936,30 @@
 		</subsets>
 	</test>
 	
+	<test>
+		<testCaseName>testStringStreams</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) $(JAVA_COMMAND) $(JVM_OPTIONS) \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+	-testnames testStringStreams \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>8</subset>
+			<subset>11</subset>
```

Comment:
Why is this only enabled for 8 and 11? It should be for all versions (i.e. 8+) so it doesn't need to be change for Java 12, for example.

Reply:
In that case, I will remove the `<subsets>` element.